app/Providers/RouteServiceProvider.php

```diff
@@ -69,13 +70,30 @@ private function mapAppRoutes()
 
     private function mapAuthRoutes(): void
     {
+        Route::middleware(['app'])
+            ->namespace($this->namespace)
+            ->prefix('auth')
+            ->group(
+                function () {
+                    //TODO: move out from 'users' - it is being used in API
+                    Route::get('users/email/confirm1Old/{token}', 'App\UsersController@emailChangeStep2');
+                    Route::get('users/email/confirm2New/{token}', 'App\UsersController@emailChangeStep3');
+                    Route::post('users/email/activate', 'App\UsersController@emailActivate');
+
+                    Route::post('login', 'App\AuthController@login');
+                }
+            )
+        ;
+
         Route::middleware(['app', 'guest'])
             ->namespace($this->namespace)
             ->prefix('auth')
             ->group(function () {
                 Route::post('recovery', 'App\AuthController@recovery');
                 Route::get('recovery/{token}', 'App\AuthController@recoveryTokenExtend');
-                Route::post('login', 'App\AuthController@login');
+
+                //TODO: move out from 'users' - it is being used in API
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1135)